gwpy/cli/timeseries.py

```diff
@@ -25,6 +25,7 @@
 from .cliproduct import TimeDomainProduct
 from ..plot import Plot
 from ..plot.tex import label_to_latex
+from astropy import units
```

Comment:
@areeda, can you please remove this import, which I don't think is needed anymore